autoconf/configure.ac

```diff
@@ -571,13 +527,12 @@ AC_SUBST([UDUNITS_EXCLUDE])
 
 dnl process the other optional command line arguments
 AX_HDF5_HOME([])
-AX_DMTCP_HOME([])
-AX_TPRO_HOME([])
-AX_BC635_HOME([])
 AX_GSL_HOME([])
 AX_GTEST_HOME([])
 AX_ER7_UTILS([])
 
-AC_OUTPUT([share/trick/makefiles/config_user.mk])
+AC_CONFIG_FILES([share/trick/makefiles/config_user.mk])
+AC_OUTPUT
+
+
 
-printf "\033@<:@32mconfigure script successfully completed\033@<:@0m\n"
```

Comment:
Is there a reason for removing this print statement?

Reply:
Nope! There is not good reason for removing the print statement. Forgot to add it back in. I'll do that.